dk.sdu.se4.screen/src/main/java/dk/sdu/se4/screen/GameScreen.java

```diff
@@ -103,7 +103,8 @@ private void drawEntitys() {
                 this.game.getBatch().draw(imagePart.getTexture(), p.getX(), p.getY());
                 this.game.getBatch().end();
             }
-
+          
+            // If true the entties collisionbox is shown. If false they dont.
             if (true) {
```

Comment:
denne if(true) er unødvendig

Reply:
den kan ikke være falsk ?
den er true uanset hvad

Reply:
Kasper har allerede forklaret dig hvorfor den er der. Det er bare noget man kan slå til / fra for at se kollisions boks.

Reply:
arh jamen kan du ikke sætten den til en tast fx G i stedet for fra eller til :)

Reply:
eller lave en //TODO  :) så kan vi lave en option i menu der slå den fra og til